engine/flink/components/base-unbounded/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/EventGeneratorSourceFactory.scala

```diff
@@ -250,6 +251,22 @@ object EventGeneratorSourceFactory
     }
   }
 
+  private def flinkSource(schedule: Duration) = {
+    new DataGeneratorSource[Void](
+      (_: java.lang.Long) => null,
+      Long.MaxValue,
+      new RateLimiterStrategy {
```

Comment:
Would `RateLimiterStrategy.perSecond(1.second / schedule)` work here? If I'm reading the source right the only difference is that `perSecond` takes parallelism into account, and our implementation has always assumed parallelism=1.